odoo/models.py

```diff
@@ -366,7 +365,6 @@ def expand_ids(id0, ids):
 READ_GROUP_ALL_TIME_GRANULARITY = READ_GROUP_TIME_GRANULARITY | READ_GROUP_NUMBER_GRANULARITY
 
 
-
```

Comment:
The diff should only contain the fix and the test.  Everything else is noise.  Please remove those "code cleaning" changes from the branch.